### PR TITLE
Bug/missing data path

### PR DESCRIPTION
**Pull request by miguel-chacon**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change
There is a bug, the directory where confluence store its data is not defined

### CODE REVIEW

File: tasks/install.yml [hunk 1]
```diff
@@ -30,12 +40,16 @@
...
-- name: Confluence | Checking confluence home path
+- name: Confluence | Checking confluence paths permissions
   file:
-    path: "{{ confluence_home_path }}"
+    path: "{{ item }}"
     state: directory
     owner: "{{ confluence_user }}"
     group: "{{ confluence_group }}"
+    recurse: yes
+  with_items:
```
Comment (jnogol) [inline tasks/install.yml]: (I'm ok with this, but just as a comment) why not `loop` instead of `with_items`?

**Comment by jnogol** [comment]
could you please add this change to the changelog?

**Comment by miguel-chacon** [comment]
> could you please add this change to the changelog?

Done

File: tasks/install.yml (hunk 1 above)
Comment (miguel-chacon) [reply tasks/install.yml]: just habits, both of them would work

**Pull request merged**